flowcontrol/bbr/limiter.go

```diff
@@ -84,9 +79,7 @@ func (l *Limiter) run(ctx context.Context) {
 			// might be in the far future, and there is no ack on its way
 			// to save us from our ignorance. Fortunately we always want
 			// to send if there's nothing on the wire, so just do it.
-			// Note that we don't use sendReqs here, since we don't want
-			// to skip over the logic that checks for app limited flows:
```

Comment:
I have no idea what my reasoning was with this comment, but it makes no sense as a justification, since either way the probability of choosing other branches of the select is non-zero.

I want to stare at this a bit more; I wonder if there's some bookkeeping that I knew we needed to be doing here, but apparently failed at actually ensuring we did? I'll have to have a look when we revisit fixing the estimation problems.